plugins/plugin-github/che-plugin-github-ext-github/src/main/java/org/eclipse/che/ide/ext/github/client/GitHubClientService.java

```diff
@@ -143,9 +162,15 @@ void getPullRequest(@NotNull String owner,
      * @param callback
      *         callback called when operation is done.
      */
-    void createPullRequest(@NotNull String user, @NotNull String repository, @NotNull GitHubPullRequestCreationInput input,
+    void createPullRequest(@NotNull String user,
+                           @NotNull String repository,
+                           @NotNull GitHubPullRequestCreationInput input,
                            @NotNull AsyncRequestCallback<GitHubPullRequest> callback);
```

Comment:
ere we just fixed the formatting, and do the same thing as with other methods (deprecated + with writing new [one](https://github.com/eclipse/che/pull/618/files#diff-42660c87c5d1fa906f8222180fd1442bR170))